python/utils/spacy_eval.ipynb

```diff
@@ -0,0 +1,169 @@
+{
+ "cells": [
```

Comment:
this notebook should also include the installation instructions for this dataset as a comment: python -m spacy download en_core_web_lg